tests/test_persisted_scale_property.py

```diff
@@ -0,0 +1,21 @@
+"Contains a collection of pytest tests that validates the persisted scale properties."
+
+import nidaqmx
+import nidaqmx.system.system
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPersistedScaleProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the persisted scale properties."""
+
+    def test_bool_property(self):
+        """Test for validating bool attributes in persisted scale."""
+        system = nidaqmx.system.System.local()
+        persisted_scale = system.scales[0]
+        assert persisted_scale.allow_interactive_editing
+
+    def test_string_property(self):
+        """Test for validating string attributes in persisted scale."""
+        system = nidaqmx.system.System.local()
+        persisted_scale = system.scales[0]
+        persisted_scale.author
```

Comment:
Here is how to validate this attribute:
- Add the `Author` attribute to one of the scales in the `.ini` file. You can edit the scale in MAX and export the `.ini` to see what the correct format is.
- Change this test to create a scale object for a specific scale by name, e.g. `nidaqmx.system.Scale("polynomial_scale")`
- Assert that the `author` property matches the `.ini` file's `Author` attribute.

Likewise, you can test a float property by checking `scale.lin_slope` for the scale named "double_gain_scale" and you can test a list[float] property by checking `scale.poly_forward_coeff` for the scale named "polynomial_scale".

Reply:
Re: `lin_scope` and `poly_forward_coeff`: I see those properties are in the `Scale` class not the `PersistedScale` class, so they would go in a different test.